google/cloud/pubsub/publisher_connection.cc

```diff
@@ -138,13 +138,13 @@ std::shared_ptr<pubsub::PublisherConnection> MakePublisherConnection(
     if (options.message_ordering()) {
       auto factory = [topic, options, sink, cq](std::string const& key) {
         return BatchingPublisherConnection::Create(
-            topic, options, key, SequentialBatchSink::Create(std::move(sink)),
-            cq);
+            std::move(topic), std::move(options), key,
+            SequentialBatchSink::Create(std::move(sink)), std::move(cq));
       };
       return OrderingKeyPublisherConnection::Create(std::move(factory));
     }
     return RejectsWithOrderingKey::Create(BatchingPublisherConnection::Create(
-        std::move(topic), std::move(options), {}, sink, std::move(cq)));
+        topic, options, {}, std::move(sink), std::move(cq)));
```

Comment:
This is called at most once, and therefore a `std::move(topic)` was appropriate.

Reply:
whoops